storage/plugins/book-club/src/LendingRepo.php

```
@@ -218,11 +218,19 @@ public function myActiveLoans(int $clubId, int $userId): array
     public function createOffer(int $clubId, int $clubBookId, int $lenderId, string $notes): ?int
     {
         $notesOrNull = $notes !== '' ? $notes : null;
+        // Atomic check-and-insert: the controller's hasOpenOffer() pre-check
+        // is only UX — two concurrent submits must not both insert, so the
+        // one-open-offer invariant is re-enforced inside the INSERT itself.
         $ok = $this->execAffected(
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $ok. Configured minimum length is 3. (undefined)

(ShortVariable)